recipes/docbook-xsl/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "DocBook XSL" %}
+{% set tarname = name|lower|replace(' ', '-') %}
+{% set version = "1.79.2" %}
+
+package:
+  name: {{ tarname }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/docbook/xslt10-stylesheets/releases/download/release/{{ version }}/{{ tarname }}-{{ version }}.tar.gz
+  sha256: 966188d7c05fc76eaca115a55893e643dd01a3486f6368733c9ad974fcee7a26
+
+build:
+  noarch: generic
+  number: 0
+
+requirements:
+  run:
+    # xmlcatalog is required by post-link.{bat,sh} and
+    # pre-unlink.{bat,sh}.
+    - libxml2
+
+test:
+  requires:
+    # Really need 2.10.3 build 6 or later.
+    - libxml2
```

Comment:
hmm this isn't really required here, but lets go with it.

Reply:
The idea really was to remember to depend on 2.10.4 or some such whenever that becomes available.

Reply:
you will want to make it a run constraint to be honest. Otherwise things like the migrator will have trouble.

Reply:
I see; I'll make a note fix this on the next update.